docs/README.md

```diff
@@ -14,6 +14,84 @@ limitations under the License.
 
 # Documentation
 
-No documentation available yet. Start your first contribution with some documentation.
+> Highly customizable open source theme for Hugo based static websites
+
+Syna is a Hugo template built to be customizable and easy to use at the same time. To achieve this we have built the theme in a way that all the pages would be created by using building blocks
+called fragments. Each page of a website using Syna is created using fragments.
+
+## Fragments
+
+Fragments are the way you define your view. There are several prebuilt fragments that are necessary to a website and users can write their own fragments and even override the default fragments. These fragments are controlled with page resources or fragments defined in a headless page called `global` which would be rendered in all pages.
```

Comment:
I would split it up.
1. Fragments are the way you build your site. Each site is made up of one or multiple fragments. These can be a navigation fragment, a content fragment and more.
2. Multiple pre-bundled fragments are available. You can add your own custom fragment... (explanation)
3. For fragments of a site, that need to show up on every page, we have global fragments. These fragments are controlled with page resources or fragments defined in a headless page called `global` which would be rendered in all pages.
You can selectively overwrite a global fragment on each page. Create a fragment with the same filename as the global one and it will be overwritten within this page.

Reply:
The numbers are just there to clarify the various parts.